schlib/rules/S6_2.py

```diff
@@ -93,12 +93,69 @@ def checkDatasheet(self):
 
         # Datasheet field must be empty
         if not self.checkEmpty(ds):
-            self.error("Datasheet field must be EMPTY")
+            self.error("Datasheet field (.lib file) must be EMPTY")
             fail = True
 
         return fail
 
-    def check(self):
+    def checkDocumentation(self, name, documentation, alias=False, isGraphicOrPowerSymbol=False):
+
+        errors = []
+        warnings = []
+
+        if not documentation:
+            errors.append("Missing all metadata information in the .dcm file (datasheet, keyword and description)")
+        elif (not documentation['description'] or
```

Comment:
Can lines 108 to 115 be combined and just check for the description or the keywords with another clause at the end for datasheet and power symbols?

Reply:
The intention is to combine the two rules with as little changes as possible in this pull request. Any further changes can be made in a separate pull request later.

Reply:
It looks like all this code is new so I'm not quite following you, but OK. Seems logically correct even if it's verbose and/or hard to follow.

Reply:
It is copied from another rule (rule 6.3 no longer exists so all the checks from that file were move in here as these two rules are combined now in the KLC. This is the downside of naming the rule check files after the rule number instead of giving them a representative name of what they actually chekc)